js/binance.js

```diff
@@ -418,7 +492,12 @@ module.exports = class binance extends Exchange {
 
     async fetchTime (params = {}) {
         const type = this.safeString2 (this.options, 'fetchTime', 'defaultType', 'spot');
-        const method = (type === 'spot') ? 'publicGetTime' : 'fapiPublicGetTime';
+        let method = 'publicGetTime';
+        if (type === 'future') {
+            method = 'fapiPublicGetTime';
+        } else if (type === 'delivery') {
+            method = 'dapiPublicGetTime';
+        }
```

Comment:
This now uses `publicGetTime` for `margin` (previous code used `fapiPublicGetTime`).

Reply:
Where are these publicGetTime implemented?